pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/MPSTNCuda.hpp

```diff
@@ -208,9 +208,13 @@ class MPSTNCuda final : public TNCudaBase<Precision, MPSTNCuda<Precision>> {
     };
 
     /**
-     * @brief Get final state of the quantum circuit.
+     * @brief Set MPS final state of the quantum circuit.
+     *
+     * @param cutoff Cutoff value for SVD decomposition. Default is 0.
+     * @param cutoff_mode Cutoff mode for SVD decomposition. Default is "abs".
      */
-    void get_final_state() {
+    void set_mps_final_state(double cutoff = 0,
```

Comment:
hmmm, I would like to change the name because `get_final_state` sounds like this method will return some data. But it returns nothing but compute the final mps state and append it to the compute graph. What about change it to `append_mps_final_state`?

Reply:
I like `append_mps_final_state` ;)